Answers.md

```diff
@@ -1,11 +1,16 @@
 # Answers
 
 1.  What is React JS and what problems does it try and solve?
+    React is a library that allows developers to create repeating user interfaces easily. These interfaces can be mixed and matched that passes in similarly typed data.
 
 1.  What does it mean to _think_ in react?
+    To think in React means to nest components using props that pass down from parent to children to grandchildren.
 
 1.  Briefly describe some of the differences between a Class/Stateful component and a Functional/Presentational component.
+    Functional components are Javascript Functions that take in props as an argument. Functional components can use arrow function, meaning we can skip the function and return keyword and omit the curly brackets. They are usually addative components to the Class component. Class componenets offer more features such as constructor and state. It is the same as a functional componenet, but more complex as it has more feautures. 

```

Comment:
You still need the return statement in functional components, but you don't need the render method.